shoop/core/models/_contacts.py

```diff
@@ -39,6 +39,12 @@ class Meta:
     def __str__(self):
         return self.safe_translation_getter("name", default="Group<%s>" % (self.identifier or self.id))
 
+    def get_nonmembers(self):
+        return Contact.objects.exclude(groups=self.id).order_by("name")
+
+    def get_members(self):
```

Comment:
I think this is not needed either. You can use contact_group.members.all().order_by("name") at the formpart.